index.html

```diff
@@ -159,85 +157,51 @@
-        <li>Validate the <i><dfn>request destination</dfn></i> given by the
-        `as` attribute. If the attribute is omitted, then initialize it to the
-        empty string. If the provided value is not a [valid request
-        destination], then [queue a task] to [fire an event] named
-        `error` at the [link] element and abort these steps.[[!FETCH]]</li>
-        <li>If the `type` attribute's value is missing or is the empty string,
-        then continue to the next step.
-        Otherwise, If the `type` attribute's value is not a [parsable MIME type]
-        or is an [unsupported MIME type], then abort these steps. </li>
-        <li>If the `media` attribute's value is missing or is the empty string,
-        then continue to the next step.
-        Otherwise, if the `media` attribute's value is not a [valid media query
-        list] that does [match the environment], then abort these steps.</li>
-        <li>Do a potentially CORS-enabled fetch of the resulting <i>absolute
-        URL</i>, with the <i>mode</i> being the current state of the element's
-        [crossorigin] content attribute, the <i>origin</i> being the [origin]
-        of the [link] element's [node document], <i>destination</i> set to
-        <i>request destination</i>, and the <i>default origin behavior</i> set
-        to <i>taint</i>.</li>
-      </ol>
-      <p>The <a>preload link</a> element MUST NOT [delay the load event] of the
-      element's [node document].</p>
-      <p class="note">The fetch initiated by a <a>preload link</a> is subject
-      to relevant CSP policies determined by the value of the `as` attribute -
-      e.g. when set to `image` the fetch is subject to `image-src`. Similarly,
-      the value of the `as` attribute is used to initialize request headers and
-      priority. If the `as` attribute is omitted, the request will be
-      initialized with similar processing and request properties as one
-      initiated by `XMLHttpRequest`.</p>
-      <p>Once a <dfn>preload resource has been <a data-lt=
-      "obtain the preload resource">obtained</a></dfn>, the user agent must run
-      these steps:</p>
+      <p>At these times, the user agent must [obtain the resource] given by the
+      link element.</p>
+      <p>Obtaining the resource given by a <a>preload link</a> element MUST NOT
+      [delay the load event] of the element's [node document].</p>
+      <p>Once a <dfn>preload resource has been obtained</dfn>, the user agent
+      must run these steps:</p>
       <ol>
-        <li>If the load was successful, [queue a task] to [fire an event]
-        named `load` at the [link] element. Otherwise, [queue a task] to [fire
-        a simple event] named `error` at the [link] element.</li>
+        <li>If the load was successful, [queue a task] to [fire an event] named
+        `load` at the [link] element. Otherwise, [queue a task] to [fire an
+        event] named `error` at the [link] element.</li>
```

Comment:
Good point. I guess it is easy to miss because it is not part of the algorithm...